.gitmodules

```
@@ -1,3 +1,9 @@
 [submodule "tests/html-ts-mode"]
 	path = tests/html-ts-mode
 	url = git@github.com:mickeynp/html-ts-mode.git
+[submodule "test/tuareg"]
+	path = test/tuareg
+	url = git@github.com:ocaml/tuareg.git
+[submodule "test/ocaml-ts-mode"]
+	path = test/ocaml-ts-mode
+	url = git@github.com:terrateamio/ocaml-ts-mode.git
```

```suggestion
[submodule "test/tuareg"]
	path = tests/tuareg
	url = git@github.com:ocaml/tuareg.git
[submodule "test/ocaml-ts-mode"]
	path = tests/ocaml-ts-mode
	url = git@github.com:terrateamio/ocaml-ts-mode.git
```

These are getting checked out to the wrong path.